config.mk

```diff
@@ -24,6 +24,13 @@ COMMIT := $(shell git describe --tags --always 2> /dev/null || echo "unknown")
 
 # setup general used CFLAGS
 CFLAGS   += -std=c99 -pipe -Wall -fPIC
+CFLAGS   += -I/usr/include/atk-1.0
```

Comment:
Is this really needed? Note you can give the CFLAGS direct to the calls of make.
```
CFLAGS="-I/usr/include/cairo -I/usr/..." make ...
```
I think the include paths should be set by the line `CFLAGS += $(shell pkg-config --cflags $(LIBS))` at the end of the _config.mk_. You could check this by calling
```
pkg-config --cflags gtk+-3.0 'webkit2gtk-4.0 >= 2.3.5'
```